cmake/wolfboot.cmake

```diff
@@ -75,15 +75,15 @@ function(gen_wolfboot_signed_image TARGET)
 
     # generate signed image
     add_custom_command(
-        OUTPUT ${TARGET}_signed.bin
-        DEPENDS ${INPUT_IMAGE} ${WOLFBOOT_SIGNING_PRIVATE_KEY}
+        OUTPUT ${TARGET}_v${VERSION}_signed.bin
```

Comment:
does changing the output binary name interfere with any automated scripts that may expect the old name without the version?

Reply:
hi @jpbland1 , no automated scripts are currently using Cmake, so they won't be affected. Also, this change seems to use the correct name format now (as in image_v1_signed.bin) for the output file of the signing operation

Reply:
@danielinux hit the nail on the head with his comment. This changes the cmake build system to use the name format actually produced by the signing tools. In our internal company fork of wolfBoot, which the cmake changes are originally based off of, I modified the signing tools to not include the version number in the output file name.